### ISSUE TITLE
1.4.13: Requirement for 'Hoverable' is unclear

### ISSUE THREAD

**Issue by jfhector** [alphagov/wcag-primer#50]
The requirements for the 'hoverable' part of [1.4.13 Content on Hover and Focus](https://alphagov.github.io/wcag-primer/all.html#1-4-13-content-on-hover-or-focus-aa) say to:
> Manage focus and move the mouse pointer directly from the trigger onto the new content.

I don't understand what 'manage focus' means in the context of making an element hoverable.
It sounds like "manage focus" refers to managing keyboard focus.

Note: I'm reporting all these issues because I believe that this doc will be very useful to many people once it's finalised.

**Comment by selfthinker** [comment]
I agree, when we discussed this in the team we couldn't make sense of it either.

Suggestion for writing it differently:

> When hovering over the trigger, move the mouse pointer over the new content. That content should stay visible.

**Comment by jfhector** [comment]
Sounds good to me.
